src/cell_classification/segmentation_data_prep.py

```diff
@@ -415,9 +415,9 @@ def check_additional_inputs(self):
 
         # CELL TYPE TABLE
         # drop all columns except cell_type_key, segment_label_key, sample_key
-        self.cell_type_table.drop(self.cell_type_table.columns.difference([
+        self.cell_type_table.drop(labels=self.cell_type_table.columns.difference([
```

Comment:
How come there's actual code changes required? Does coveralls fail on things that are passing pytest?

Reply:
The tests don't pass without these changes, due to Pandas (probably a Pandas 2.0 deprecation) not being able to understand certain TensorFlow data types, specifically `tf.bool`.

Reply:
But the tests were passing before? This was just for coveralls, right? Or you're saying even the general GHA tests were failing

Reply:
Looks like tests were passing before in #55 even though it was using `Pandas` 2.0. I cannot figure out why, for this PR, they did not pass. Looking at this [test workflow](https://github.com/angelolab/cell_classification/actions/runs/4723965695/jobs/8380598241#step:5:1175) shows an error.

Coveralls was not working properly because the `.coveragerc` file conflicted with the pytest configuration in the `pyproject.toml` file.